Use standard names of remotes.

vkucera opened the pull request:
Related PRs:
* https://github.com/alisw/alibuild/pull/691
* https://github.com/alice-doc/alice-analysis-tutorial/pull/128

vkucera commented:
@ktf

ktf commented:
I merged the other PR in alibuild, but then again, maybe people will complain. We should at least discuss the change in the computing board and in WP3, because IIRC we already had this discussion 6 years ago and the rationale was: "origin and upstream are confusing names, let's use `<username>` and `<origin>`.

@Barthelemy @jgrosseo @pzhristov do you remember any decision about this?

vkucera commented:
> I merged the other PR in alibuild, but then again, maybe people will complain. We should at least discuss the change in the computing board and in WP3, because IIRC we already had this discussion 6 years ago and the rationale was: "origin and upstream are confusing names, let's use `<username>` and `<origin>`.
> 
> @Barthelemy @jgrosseo @pzhristov do you remember any decision about this?

I dare to strongly disagree with the quoted rationale.

`upstream` and `origin` are standard names with clear meaning, consistently referred in the GitHub tutorials and elsewhere on the Internet:
* https://help.github.com/en/articles/configuring-a-remote-for-a-fork
* https://help.github.com/en/articles/syncing-a-fork
* https://www.atlassian.com/git/tutorials/setting-up-a-repository

Deviating from the standard nomenclature (which is the relevant one whenever someone tries to google out some help) by calling the fork remote `<username>` and the forked remote `origin` creates much more confusion and is actually counterproductive and damaging.

vkucera commented:
Rebased with master and fixed duplicate links to the Analysis tutorial.
@ktf It would be useful if all the three related PRs could be merged together to avoid temporary discrepancies.

vkucera commented:
@ktf https://github.com/alice-doc/alice-analysis-tutorial/pull/128 has been already merged. Could you merge this one too?
